internal/liveness/metrics_util.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2018 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package liveness
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/ceph/ceph-csi/internal/util"
+	"k8s.io/klog"
+)
+
+const (
+	// volIDVersion is the version number of volume ID encoding scheme
+	volIDVersion uint16 = 1
+
+	// csiConfigFile is the location of the CSI config file
+	csiConfigFile = "/etc/ceph-csi-config/config.json"
+
+	// RADOS namespace to store CSI specific objects and keys
+	radosNamespace = "csi"
+)
+
+// CheckImageInCluster check if image in cluster
+func CheckImageInCluster(imageName string, imagePool string, monitors string, cr *util.Credentials) bool {
```

Comment:
already we have a function to get rbd image info

Reply:
done